package/endpoint_adapter/src/endpoint_adapter.c

```diff
@@ -99,6 +99,7 @@ typedef ssize_t (*write_fn_t)(handle_t *handle, const void *buffer,
                               size_t count);
 
 bool debug = false;
+static bool name_set = false;
```

Comment:
I would think we could change the initial value of `port_name` to `NULL` and just check that directly instead of having to create another flag variable. Examples of this in other daemons:

https://github.com/swift-nav/piksi_buildroot/blob/e47e15952ab3c37e28497b14806f8bc2f38198f3/package/skylark_daemon/src/skylark_daemon.c#L34

https://github.com/swift-nav/piksi_buildroot/blob/e47e15952ab3c37e28497b14806f8bc2f38198f3/package/skylark_daemon/src/skylark_daemon.c#L153-L156

Reply:
👍